course_discovery/populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import datetime
+from dateutil.parser import parse
```

Comment:
I'm fairly certain this is a third-party import, and should be grouped appropriately.